src/routes/signin/passwordless/email.ts

```diff
@@ -117,7 +114,8 @@ export const signInPasswordlessEmailHandler: RequestHandler<
       link,
       displayName: user.displayName,
       email,
-      newEmail: user.newEmail,
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      newEmail: user.newEmail!,
```

Comment:
How about we check `user.newEmail` in an if statement before and throw if it's unavailable? 

Feels cleaner than adding an eslint comment.

Reply:
This was due to incorrect types, as `newEmail` is only mandatory when changing the email. II improved the template typings.